tasks/data_handling/data_pre_processing.py

```diff
@@ -0,0 +1,149 @@
+import scanpy as sc
+
+def create_count_matrix(file_path : str = None) -> sc.AnnData:
+    """
```

Comment:
We need to standardize comments. I don't mind the format, but then everything we write has to be in the same format. So, two options here: 1) Make this format the standard. Go back to what we already have, and reformat the comments there. Make a pull request for the other one. And then we'll merge this PR. 2) Take the comments of what we already have as the standard and reformat current code.

Reply:
I prefer this format since it shows up when you use the function so I'll change the other comment!

Reply:
Perfect! Please post the PR for the other change (for which you need a new branch) as a reply here so I can close it :)